PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -85,31 +85,41 @@ intptr_t CALLBACK AboutDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM lPar
 		case NPPM_INTERNAL_REFRESHDARKMODE:
 		{
 			NppDarkMode::autoThemeChildControls(_hSelf);
+			if (_hIcon != nullptr)
+			{
+				::DestroyIcon(_hIcon);
+				_hIcon = nullptr;
+			}
 			return TRUE;
 		}
 
-		case WM_DRAWITEM :
+		case WM_DPICHANGED:
 		{
-			DPIManager& dpiManager = NppParameters::getInstance()._dpiManager;
-			int iconSideSize = 80;
-			int w = dpiManager.scaleX(iconSideSize);
-			int h = dpiManager.scaleY(iconSideSize);
+			DPIManagerV2::setDpiWP(wParam);
+			destroy();

```

Comment:
I suppose `WM_DPICHANGED` message is sent before `WM_DRAWITEM`, so I see your logic to call the destroy method. However, in destroy() method, not only the `_hIcon` is destroyed, but also `_pageLink`.

Reply:
`_pageLink` has old font size.

Reply:
My point is _hIcon is destroyed in WM_DPICHANGED, but it will be recreated in WM_DRAWITEM.
Whereas it's not the case for _pageLink. Or am I missing somethings?

Reply:
IMO, here we should use 
```cpp
if (_hIcon != nullptr)
{
	::DestroyIcon(_hIcon);
	_hIcon = nullptr;
}
```
instead of `destroy()`